QModManager/Patching/ManifestValidator.cs

```diff
@@ -62,10 +62,10 @@ public void ValidateManifest(QMod mod)
                     mod.SupportedGame = QModGame.Subnautica;
                     break;
                 default:
-                {
-                    mod.Status = ModStatus.FailedIdentifyingGame;
-                    return;
-                }
+                    {
+                        mod.Status = ModStatus.FailedIdentifyingGame;
+                        return;
+                    }
```

Comment:
Are these brackets supposed to be here after default?

Reply:
It's just a code style preference that's ultimately just about whitespace/indenting and doesn't really affect anything.

Might as well revert this change since it was probably done by accident.

Reply:
Yeah I have a habit of hitting "format" whenever I save... sorry about that.

Reply:
da8345d